webpack/scenes/FlatpakRemotes/CreateEdit/FlatpakRemoteform.js

```diff
@@ -68,12 +70,19 @@ const FlatpakRemotesForm = ({ setModalOpen, remoteData }) => {
 
   const onSave = () => {
     setSaving(true);
-    dispatch(createFlatpakRemote({
-      name,
-      url,
-      username,
-      password,
-    }));
+    dispatch(isEditing ?
+      updateFlatpakRemote(editingId, {
+        name,
+        url,
+        username,
+        password,
```

Comment:
```suggestion
        token: password,
```
The params we pass to the API have token as the password and is called token so we need to map it.

Reply:
We'll need to work a little to support password editing cause currently we don't send password over the API so it always shows up empty on the edit form and is sent back as empty to API clearing out password unintentionally.

We have patterns like this which we'll need to use. Something that we do for Alternate content sources like this: 
https://github.com/Katello/katello/blob/14e1906cda0b473075f395ec67ab1e107cc2b85c/webpack/scenes/AlternateContentSources/Details/EditModals/ACSEditCredentials.js#L34